services/blockchain-indexer/shared/dataService/index.js

```diff
@@ -108,7 +104,9 @@ const { getGenerators } = require('./generators');
 const { invokeEndpoint } = require('./invoke');
 
 const {
+	getNFTs,
```

Comment:
Can we please rename the method to `getNFT` instead?
```suggestion
	getNFT,
```